### PR TITLE
adding support for docker

### PR DESCRIPTION
**Pull request by ctava**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -208,6 +208,21 @@ And then:
...
+    [Install Docker](https://docs.docker.com/engine/installation/#supported-platforms)
+    git clone this repo:
+    `git clone https://github.com/shenwei356/seqkit`
+
+    Run the following commands:
+    ```
+    cd seqkit
+    docker build -t shenwei356/seqkit .
+    docker run -it -p 8888:8888 shenwei356/seqkit:latest
+    go build
```
Comment (shenwei356) [inline README.md]: `go get -u github.com/shenwei356/seqkit/seqkit` has built seqkit, and seqkit is ready to run. 

For test:

    cd /go/src/github.com/shenwei356/seqkit
    ./tests/test.sh

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ENV PATH $PATH:$GOPATH/bin:/usr/local/go/bin
+RUN curl -fsSL "$GOLANG_DOWNLOAD_URL" -o golang.tar.gz && \
+    echo "$GOLANG_SHA256_CHECKSUM golang.tar.gz" | sha256sum -c - && \
+    sudo tar -C /usr/local -xzf golang.tar.gz && \
+    rm golang.tar.gz && \
+    mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
+#End: install golang
+
+#Begin: install delve
+RUN go get github.com/derekparker/delve/cmd/dlv
```
Comment (shenwei356) [inline Dockerfile]: What's dlv used for? It seems not used.

**Comment by shenwei356** [comment]
Hi Chris, thanks for this PR. 

Seqkit provides executable binary files for Win/Linux/OS X, which is very easy to install, do we really need to install and start seqkit in docker? What's your application environment?

File: Dockerfile (hunk 2 above)
Comment (ctava) [reply Dockerfile]: dlv is a debugger. add it as a utility: 
https://github.com/derekparker/delve

File: README.md (hunk 1 above)
Comment (ctava) [reply README.md]: when executing ./tests/test.sh binary could not be found

**Comment by ctava** [comment]
Hi Wei - 

application environment is : 
MacOS (laptop) - developer environment
amazon linux

ssshtest uses wget. sure can brew install wget --with-libressl
but the point of docker is to provide a standard application environment irrespective of the operation system of the host

File: README.md (hunk 1 above)
Comment (shenwei356) [reply README.md]: I tested on a fresh vps, it can be found, cause

    ENV PATH $PATH:$GOPATH/bin:/usr/local/go/bin

**Pull request merged**